basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/conceptdescriptionrepository/http/TestConceptDescriptionRepositoryHTTP.java

```diff
@@ -161,6 +161,16 @@ public void updateNonExistingConceptDescription() throws IOException {
 
 		assertEquals(HttpStatus.NOT_FOUND.value(), updateResponse.getCode());
 	}
+	
+	@Test
+	public void updateConceptDescriptionWithMismatchId() throws IOException, ParseException {
+		String id = ConceptDescriptionRepositorySuiteHelper.CONCEPT_DESCRIPTION_ID;
+		String expectedConceptDescriptionJSON = getUpdatedConceptDescriptionWithMismatchIdJSON();
```

Comment:
See comment on other HTTP test, this is not the expected JSON

Reply:
Done